package/yast2-apparmor.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Jul 17 07:25:37 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Avoid warnings due to dead code related to notify settings,
+  which were removed some time ago (bsc#1139852).
+- Removes a Perl script to generate reports that is broken
+  and is not used for a long time ago.
+- 3.2.1.1
```

Comment:
Hmmm, I have just realized there was no need to add an extra digit in the original pull request. 3.2.2 is not taken in any branch. It would have been ok to merge this in SP3, SP4 and SP5 as 3.2.2, as far as I can see.

Reply:
Ok. Let me fix the other one first :)